game-core/src/main/java/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -67,9 +67,6 @@
   // territories where a battle occurred
   // TODO: fix typo in name, 'fough' -> 'fought'
   private final Set<Territory> m_foughBattles = new HashSet<>();
-  // these territories have had battleships bombard during a naval invasion
-  // used to make sure that the same battleship doesn't bombard twice
-  private final Set<Territory> m_bombardedFromTerritories = new HashSet<>();
```

Comment:
Are you sure this is gonna be save compatible?

Reply:
Ditto other comment.